source/manual/alerts/email-alert-api-high-queue-latency.html.md

```diff
@@ -5,19 +5,38 @@ section: Icinga alerts
 subsection: Email alerts
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-07-21
+last_reviewed_on: 2020-08-05
 review_in: 6 months
 ---
 
 ### High queue latency (queue_latency)
 
-This means the time it takes for a job to be processed is unusually high. This
-may indicate a problem down the line which is preventing workers from being
-processed. It may also imply the threshold is too low if a large number of
-emails have being sent out due to a content change.
+This means the time it takes for a job to be processed is unusually high. See
+our [Sidekiq queue latency] graph for more detail.
 
-See the [sidekiq][sidekiq] section for more information about the Sidekiq queues,
-or read [email troubleshooting].
+Some diagnostic steps you could take are below:
+
+* Check the [Sidekiq queue size] graph to see if we have a high number of emails
+  queued up.
+* If there is a high amount of emails in the queue, are we processing them at a
+  normal speed? Under high load we regularly send requests to Notify at a rate
+  of ~300 email requests per second. To check this, look at our [email send
+  attempts] graph.
+* Are the workers reporting any problems or any issues being raised in
+  [Sentry]?
+* Check [Kibana] to see our Sidekiq logs for Email Alert API.
+
+See the [Sidekiq] section for more information about Sidekiq or read
+[email troubleshooting].
+
+For more information on the individual queues and what they do, see
+[here][email_queues].
 
 [email troubleshooting]: /manual/email-troubleshooting.html
-[sidekiq]: /manual/sidekiq.html
+[Sidekiq]: /manual/sidekiq.html
+[Sentry]: https://sentry.io/organizations/govuk/issues/?project=202220&statsPeriod=12h
+[Sidekiq queue size]: https://grafana.blue.production.govuk.digital/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=email-alert-api&var-Queues=All&from=now-3h&to=now&panelId=1&fullscreen
```

Comment:
Minor: we could just link to the dashboard, so it's clear both graphs are in the same place.

Reply:
Sure